### PR TITLE
FIB 1.2 Documentation Update

### PR DESCRIPTION
**Pull request by yamkovoy**
Closes issue #47

### CODE REVIEW

File: src/overview/tokenomics.md [hunk 1]
```diff
@@ -32,14 +32,30 @@ The number of digital credits that a peer has on their account, and that can be
...
+
+## Current default conversion rate
 
 - **Uploading fraud events**: `10` credits per `id` fraud identifier.
 - **Downloading fraud events**: `1` credit per `id` fraud identifier.
 
 ## Welcome bonus
 
-Every new peer that has successfully registered with FIB, automatically receives a 1000 credits bonus.
+Every new peer that has successfully registered with FIB, automatically receives a `1,000` credits bonus.
```
Comment (0x009922) [inline src/overview/tokenomics.md:58]:
```suggestion
Every new peer that has successfully registered with FIB, automatically receives a $1\ 000$ credits bonus.
```

You can use [Math Equations](https://vitepress.dev/guide/markdown#math-equations) syntax if you need. **Note:** you should enable it (available after #58 merging) with `markdown.math = true` in `config.mts`. Refer to the link.

Example render: $1\ 000$

**Review by 0x009922 — APPROVED** [review]

File: src/overview/tokenomics.md (hunk 1 above)
Comment (Mingela) [reply src/overview/tokenomics.md:58]: I'm ok with either of the options

File: src/api-specification/contribution-controller/retrieving-top-contributions.md [hunk 2]
```diff
@@ -17,7 +17,8 @@
...
-| `fetch-mode` | `string` | The parameter that filters the response entries based on whether the requesting user has already seen them. <br> Could be one of the following: <ol><li>`NEW` — includes only the unseen by the requesting user entries.</li></ol><li>OLD — includes only the already seen by the requesting user entries.</li><li>`DEFAULT` — includes all of the entries.</li> <br> The `fetch-mode` parameter is set to `DEFAULT` by default. |
+| `fetch-mode` | `string` | The parameter that filters the response entries based on whether the requesting user has already seen them. <br> Could be one of the following: <ol><li>`NEW` — includes only the unseen by the requesting user entries.</li><li>`OLD` — includes only the already seen by the requesting user entries.</li><li>`DEFAULT` — includes all of the entries.</li></ol> <br> The `fetch-mode` parameter is set to `DEFAULT` by default. |
+| `confidence-score` | `boolean` | A boolean that defines whether the price of the retrieved contributions is affected by the [confidence index](../../overview/tokenomics.md#confidence-index). <br> The `confidence-score` parameter is set to `false` by default. |
```
Comment (Mingela) [inline src/api-specification/contribution-controller/retrieving-top-contributions.md:21]: it seems we miss a description of the fact that contributions with a confidence index assigned are expected to be more expensive that the ones without an index
`price affected` does not give enough clarity imo

Comment (yamkovoy) [reply src/api-specification/contribution-controller/retrieving-top-contributions.md:21]: Added to the "Tokenomics" and "Retrieving top contributions" topics

**Review by Mingela — DISMISSED** [review]

**Review by Mingela — APPROVED** [review]

**Pull request merged**